pkg/apis/networking/types.go

```diff
@@ -48,30 +48,44 @@ const (
 
 // NetworkPolicySpec provides the specification of a NetworkPolicy
 type NetworkPolicySpec struct {
-	// Selects the pods to which this NetworkPolicy object applies. The array of
-	// ingress rules is applied to any pods selected by this field. Multiple network
-	// policies can select the same set of pods. In this case, the ingress rules for
-	// each are combined additively. This field is NOT optional and follows standard
-	// label selector semantics. An empty podSelector matches all pods in this
-	// namespace.
+	// Selects the pods to which this NetworkPolicy object applies.
+	// A network flow is bidirectional but initiated in one direction,
+	// from a source to a destination.
+	// The whole flow is allowed or not depending on whether the initiation is allowed.
```

Comment:
"initiation" is not a standard term and it's not obvious here that you're using it to mean both the sending side and the receiving side of the connection establishment process, as opposed to just the sending side.

The phrasing that people normally use is that NetworkPolicy applies to connections, not to packets

Reply:
Do you think that if I simply say "connection" that the reader will understand that this applies to UDP as well?